assets/src/edit-story/components/helpCenter/useHelpCenter/index.js

```diff
@@ -208,6 +223,20 @@ export function useHelpCenter() {
       ? store.state.navigationIndex
       : nextRenderState.navigationIndex;
 
+  useEffect(() => {
+    if (
+      navigationIndex > -1 &&
+      store.state.navigationFlow.length >= navigationIndex
+    ) {
+      const currentTip = store.state.navigationFlow[navigationIndex];
+      trackEvent('help_center_read_tip', 'editor', currentTip, null, {
+        unread_count: store.state.unreadTipsCount,
+      });
+    }
+    // Disable reason: avoid sending duplicate tracking events.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [navigationIndex]);
```

Comment:
@littlemilkstudio Love the new `unreadTipsCount`!

If you have any thoughts on whether this effect looks reasonable, please LMK :-)